ember-cli-build.js

```diff
@@ -1,8 +1,8 @@
 /* eslint-env node */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
+const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
-  var app = new EmberAddon(defaults, {
+  const app = new EmberAddon(defaults, {
```

Comment:
Because it is less mental overhead every time we have to do diffs during an ember-cli upgrade and because this will be the only repo that is different from the rest.  We're trying to get standardization across the repos.  If this is a change we want to roll with (and I'm not necessarily against it) then we need to do it as a concentrated effort.  At the very least, if you feel strongly about making this change at this moment in this repo then you need to open issues in all the other repos to address the same so the difference between the repos, and the desire to reconcile them, is captured.